src/nix/profile-list.md

```diff
@@ -6,26 +6,48 @@ R""(
 
   ```console
   # nix profile list
-  0 flake:nixpkgs#legacyPackages.x86_64-linux.spotify github:NixOS/nixpkgs/c23db78bbd474c4d0c5c3c551877523b4a50db06#legacyPackages.x86_64-linux.spotify /nix/store/akpdsid105phbbvknjsdh7hl4v3fhjkr-spotify-1.1.46.916.g416cacf1
-  1 flake:nixpkgs#legacyPackages.x86_64-linux.zoom-us github:NixOS/nixpkgs/c23db78bbd474c4d0c5c3c551877523b4a50db06#legacyPackages.x86_64-linux.zoom-us /nix/store/89pmjmbih5qpi7accgacd17ybpgp4xfm-zoom-us-5.4.53350.1027
-  2 flake:blender-bin#packages.x86_64-linux.default github:edolstra/nix-warez/d09d7eea893dcb162e89bc67f6dc1ced14abfc27?dir=blender#packages.x86_64-linux.default /nix/store/zfgralhqjnam662kqsgq6isjw8lhrflz-blender-bin-2.91.0
+  Index:              0
+  Flake attribute:    legacyPackages.x86_64-linux.gdb
+  Original flake URL: flake:nixpkgs
```

Comment:
Maybe these two lines could be merged into one `flake:nixpkgs#legacyPackages.x86_64-linux.gdb` ? That would make the output easier to read imho

Reply:
and makes it easy to copy-paste

Reply:
They're not necessarily the same, since flake attributes are subject to the attribute prefix search path. E.g. `spotify` resolves to `legacyPackages.x86_64-linux.spotify`. What `spotify` resolves to could change when upgrading to a new version of the flake (e.g. it could become `packages.x86_64-linux.spotify`).

Also, `flakeref#attr` is easier to cut&paste.